Garage 2.0/Controllers/VehiclesController/VehiclesController.cs

```diff
@@ -12,12 +12,15 @@ namespace Garage_2._0.Controllers.VehiclesController
     public class VehiclesController : Controller
     {
         private readonly GarageVehicleContext _context;
-
         Vehicle[] parkingSpots;
-
-        public VehiclesController(GarageVehicleContext context)
+        IConfiguration _config;
+        public VehiclesController(GarageVehicleContext context,IConfiguration config)
         {
             _context = context;
+            _config = config;
+            Global.Garagecapacity = _config.GetValue<int>("GarageCapacity:Capacity");
```

Comment:
Is it possible to have the code after the "=" in the Global class inted?

Reply:
global is static class so you cant initialize config there